index.html

```diff
@@ -22,104 +22,305 @@
           </tr>
         </thead>
         <tbody>
-          <tr>
+          <tr class="course">
             <td>1</td>
             <td><input type="text" class="course-code" /></td>
-            <td><input type="text" class="course-unit" onkeyup="Sum();" /></td>
-            <td><input class="score" type="text"  onkeyup="Sum();"/></td>
-            <td><input type="text" class="grade" onkeyup="Sum();"   readonly/></td>
-            <td><input type="text" class="point" onkeyup="Sum();"  readonly/></td>
+            <td>
+              <input
+                type="text"
+                class="course-unit"
+                onkeyup="onUnitInserted(event);"
+              />
+            </td>
+            <td>
+              <input
+                class="score"
+                type="text"
+                onkeyup="onScoreInserted(event);"
+              />
+            </td>
+            <td>
+              <input type="text" class="grade" readonly />
+            </td>
+            <td>
+              <input type="text" class="point" readonly />
+            </td>

```

Comment:
Most of these changes come from prettier. I only added a `.course` class to each table row that holds course data and changed the `onkeyup` handlers to pass the `event` and call different functions for unit and score inputs.